client/schema/v0/index.js

```diff
@@ -325,6 +353,7 @@ export default {
           },
           animals: {
             title: 'Type of animals',
+            repeats: 'species',
```

Comment:
What's going on here?

This seems pretty wildly out of scope.

Reply:
it was already in @joefitter's commit, I assume it's a fix for something else?